tests/Maintenance/GenerateMissingIdentifiersTest.php

```diff
@@ -4,24 +4,64 @@
 
 namespace ProfessionalWiki\PersistentPageIdentifiers\Tests\Maintenance;
 
-use MediaWiki\Tests\Maintenance\MaintenanceBaseTestCase;
+use Maintenance;
 use ProfessionalWiki\PersistentPageIdentifiers\Maintenance\GenerateMissingIdentifiers;
+use ProfessionalWiki\PersistentPageIdentifiers\Tests\PersistentPageIdentifiersIntegrationTest;
 
 /**
  * @covers \ProfessionalWiki\PersistentPageIdentifiers\Maintenance\GenerateMissingIdentifiers
  * @group Database
  */
-class GenerateMissingIdentifiersTest extends MaintenanceBaseTestCase {
+class GenerateMissingIdentifiersTest extends PersistentPageIdentifiersIntegrationTest {
```

Comment:
`MaintenanceBaseTestCase` is a subclass of `MediaWikiIntegrationTestCase` anyway and adds only minimal boilerplate. Using our own base class gives us access to our helper methods.